### ISSUE TITLE
Viqeo Studio as 3rd party creation tool

### ISSUE THREAD

**Issue by nixtka** [ampproject/amp.dev#4802]
### Story creation resource name

Viqeo Studio

### Type

Use [x] to mark a selection:

- [x] Story Creation Tool
- [ ] CMS
- [x] Platform
- [ ] Other

### URL

https://viqeo.tv
https://studio.viqeo.tv

### Image

https://www.dropbox.com/s/8fixocjidos4xcy/Stories-screenshot.png?dl=0

### Description

Viqeo is the all-in-one video platform for creating Videos, Web Stories, Lessons and Onboardings.

### List of Stories

https://amphtml.viqeo.tv/story?id=40574
https://amphtml.viqeo.tv/story?id=41211
https://amphtml.viqeo.tv/story?id=136
https://amphtml.viqeo.tv/story?id=40575
https://amphtml.viqeo.tv/story?id=39755
https://amphtml.viqeo.tv/story?id=41219
https://amphtml.viqeo.tv/story?id=40588
https://amphtml.viqeo.tv/story?id=40593
https://amphtml.viqeo.tv/story?id=1049
https://amphtml.viqeo.tv/story?id=40595

**Comment by matthiasrohmer** [comment]
@shishirm, could you please review this? Thanks in advance!

**Comment by nixtka** [comment]
@shishirm we have free registration and basic Stories editor, so you can register right in the platform, but if it more convenient for you we can send you invite to test website with stories feed.

**Comment by shishirm** [comment]
Hello. Thanks so much for submitting this request. We'll start the review this week and will target completion by the end of the week.

**Comment by shishirm** [comment]
Hello,

Thanks again for submitting your tool to be listed on amp.dev. Before we get into specific feedback, I just wanted to note that most of the Stories follow pre-determined templates and layouts. My understanding is that the tool is not a WYSIWYG style editor. We have other similar templated tools which we've listed as "Templated Editor". Is this an accurate classification?

Now we've provided feedback in two categories. Must-Fix feedback which are things you will need to fix and we will verify before we can list the tool on amp.dev. Nice-To-Have/Suggestions which are recommendations on how to improve the Stories and the tools to provide the best user experience.

While the feedback are provided on the Stories themselves, the hope is to implement them at the tool level such that they can be applied to all created Stories.

Must-Fix

- poster-portrait-src is not the right dimensions. These should be at least 640px by 853 px and should retain the 3:4 aspect ratio. You can find more details [here](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)).

Nice-To-Have/Suggestions

- Provide descriptive alt text for images as per [here](https://blog.amp.dev/2020/02/12/seo-for-amp-stories/).
- If videos contain burned in text, provide the exact copy of the text in the title attribute for the video element.
- Preload the AMP runtime as per [here](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/optimize_amp/#optimize-the-amp-runtime-loading).
- Provide the ability to set schema.org style Structured Data for Stories as per [here](https://blog.amp.dev/2020/02/12/seo-for-amp-stories/).
- Ensure that auto-advance is not detracting from the Story consumption experience. There were instances where the pages moved forward too quickly.
- Some videos are choppy and slow to load which indicates some optimizations needed to ensure even users on slow connections/devices get the best experience.
- Provide subtitles for videos with voice over.

**Comment by nixtka** [comment]
@shishirm thanks for the detailed feedback! Yes, "Templated Editor" is a good name for our studio, as we propose a slightly customisable templates. We will back quite soon with updated the most important changes/additions.

**Comment by nixtka** [comment]
@shishirm fixed poster-portrait-src dimensions.
And now we are doing some optimisations according your recommendations. Can you please review and add Viqeo as a templated tool please? :) Thank you in advance.

**Comment by nixtka** [comment]
@shishirm May I kindly ask you to re-revise Viqeo tool with the posters update? We would be happy to be included as a 3rd party tool to amp.dev :)

**Comment by nixtka** [comment]
@shishirm @matthiasrohmer can we please move on here? Thank you in advance!

**Comment by nixtka** [comment]
@sdenaro , I saw you few days ago approving another tool. Maybe you can help us with our issue? We can't move forward for 1,5 months after fixing only one critical comment. We've already released some of recommended features and will release support of amp-story-auto-ads next week, but still can't be noticed as a Web Stories Creation Tool. Thank you in advance!

**Comment by shishirm** [comment]
Hello,

Apologies for the delay here! I'll re-review these today and if everything
looks good, will spin up a PR for this.

**Comment by shishirm** [comment]
Hi @nixtka,

We've reviewed your Stories and it appears that they are still being flagged for incorrect poster images. While you've followed the aspect ratio guidelines, these are not the minimum dimensions. I'm copying the exact guidelines from [here](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)) below,

- Each poster image should meet the recommended minimium size:
-- Portrait: 640px x 853px
-- Landscape: 853px x 640px
-- Square: 640px x 640px
- The poster image should be in 3x4 aspect ratio for portrait, 4x3 for landscape, and 1x1 for square.

The suggestion is to build these safeguards directly into the tool instead of updating these individual Stories. These are the specific Stories that still have issues.

- https://amphtml.viqeo.tv/story?id=40575
- https://amphtml.viqeo.tv/story?id=41219
- https://amphtml.viqeo.tv/story?id=1049
- https://amphtml.viqeo.tv/story?id=40574
- https://amphtml.viqeo.tv/story?id=40588
- https://amphtml.viqeo.tv/story?id=40593
- https://amphtml.viqeo.tv/story?id=40595
- https://amphtml.viqeo.tv/story?id=136

Could you kindly update?

Also wanted to let you know of the new [Web Story Developer Preview Group](https://groups.google.com/a/amp.dev/g/web-stories-devpreview). We'll be using this forum to disseminate early previews of testable / working code for new and upcoming features in Web Stories, surveys, etc. It should be open to sign up. Please let me know if you run into any issues.

Thanks a lot.

**Comment by nixtka** [comment]
@shishirm now we've updated all previously created stories. Sorry for that. We fixed this issue more than a month ago, but previously created stories were cashed in CDN. Now we've updated all of them. Please check again.

**Comment by nixtka** [comment]
Also I've subscribed to the group, thank you for the info!

**Comment by nixtka** [comment]
@shishirm can you please re-check? Now everything works fine.

**Comment by nixtka** [comment]
@shishirm @matthiasrohmer  can we move here before the Christmas? Pleeease :_)

**Comment by shishirm** [comment]
https://github.com/ampproject/amp.dev/pull/5132 filed to fix this. Given the holidays, this might not get merged in / deployed until the new year.

**Comment by patrickkettner** [comment]
should be live! merry christmas, @nixtka

**Comment by nixtka** [comment]
@patrickkettner @shishirm thank you guys! Merry Christmas!
